src/frontend/js/widgets/CourseProductItem/components/PaymentButton/index.tsx

```diff
@@ -224,8 +225,7 @@ const PaymentButton = ({ product, billingAddress, creditCard, onSuccess }: Payme
 
   return (
     <div className="payment-button">
-      <button
```

Comment:
Sale buttons are currently displayed with Hind typography but before there were displayed with Montserrat. WDYT to customize cunningham button font family to Montserrat ?

Reply:
I didn't find any token for button's `font-size` [in cunningham](https://github.com/openfun/cunningham/blob/main/packages/react/src/cunningham-tokens.css#L198-L204).

Also, i realize that our [tokens.css in richie](https://github.com/openfun/richie/blob/master/src/frontend/scss/vendors/css/cunningham-tokens.css) have fewer entries (118L in richie vs 204L in cunningham)

Reply:
Where did you see a token file of 204L in Cunningham ?

Reply:
I realized that button component within Cunningham has no `font-family` set. I think that's why in Richie, some buttons are using Montserrat and other hind...